website/pages/docs/configuration-files.mdx

```diff
@@ -53,9 +53,9 @@ expandProps: false
 
 ## SVGO
 
-The recommended way to configure SVGO for SVGR is to use [`.svgo.yml`](https://github.com/svg/svgo/blob/master/.svgo.yml). [Several formats are supported](./packages/core/src/plugins/svgo.js) and it is relative to the transformed SVG file.
+The recommended way to configure SVGO for SVGR is to use [`svgo.config.js`](https://github.com/svg/svgo/blob/main/README.md#configuration).
 
-Even if it is not recommended, you can also use `svgoConfig` option to specify your SVGO configuration. `svgoConfig` has precedence on `.svgo.yml`.
+Even if it is not recommended, you can also use `svgoConfig` option to specify your SVGO configuration. `svgoConfig` has precedence on `svgo.config.js`.
```

Comment:
I did not check this for truth BTW, just assumed it was still true despite the `.yml` deprecation.

Reply:
It is.